test/runtests.jl

```diff
@@ -48,24 +45,25 @@ Base.summary(A::Base.ReshapedArray) = summary_build(A)
 
 a = rand(3,5,7)
 v = view(a, :, 3, 2:5)
-@test summary(v) == "3×4 view(::Array{Float64,3}, Colon(), 3, 2:5) with element type Float64"
+view_name = "view(::Array{Float64,3}, $(v.indexes[1]), 3, 2:5)"
+@test summary(v) == "3×4 $view_name with element type Float64"
 
 c = reshape(v, 4, 3)
 str = summary(c)
-@test str == "4×3 reshape(view(::Array{Float64,3}, Colon(), 3, 2:5), (4,3)) with element type Float64"
+@test str == "4×3 reshape($view_name, $((4, 3))) with element type Float64"
 
 a = reshape(1:24, 3, 4, 2)
 b = Base.PermutedDimsArrays.PermutedDimsArray(a, (2,3,1))
 str = summary(b)
 intstr = string("Int", Sys.WORD_SIZE)
-@test str == "4×2×3 permuteddimsview(reshape(::UnitRange{$intstr}, (3,4,2)), (2,3,1)) with element type $intstr"
+@test str == "4×2×3 permuteddimsview(reshape(::UnitRange{$intstr}, $((3, 4, 2))), $((2, 3, 1))) with element type $intstr"
```

Comment:
Took me a while to figure out how that passes tests on 0.5---pesky `$` was hiding in plain sight. Very clever!